### ISSUE TITLE
Training issue: The robot keeps spinning in circles

### ISSUE THREAD

**Issue by Ethan0207** [reiniscimurs/DRL-robot-navigation#194]
**Describe the issue**
Hi, I replaced the robot model and started training. The robot kept spinning in circles and couldn't converge after 40 epochs. I adjusted the random seed value from 0 to 10, but none of them worked well. I wanted to adjust the learning rate, but I couldn't find where to do it. I would like to ask how to adjust and configure the learning rate. Is it related to some specific examples?

**Provide information about your setup:**
 - OS:20.04
 - ROS version:ROS Noetic

**Comment by reiniscimurs** [comment]
Hi,

You can pass in a learning rate when initializing optimizers:
https://docs.pytorch.org/docs/stable/generated/torch.optim.Adam.html

https://github.com/reiniscimurs/DRL-robot-navigation/blob/017ed127928bb14532f15467498655f0c5b16600/TD3/train_velodyne_td3.py#L97
https://github.com/reiniscimurs/DRL-robot-navigation/blob/017ed127928bb14532f15467498655f0c5b16600/TD3/train_velodyne_td3.py#L103
